### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by kovpaka**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -347,38 +374,143 @@
...
+}
+
+.subscribe-block__title-description {
+    margin-bottom: 30px;
+    color: var(--color4);
+}
+
+.subscribe-block__input {
+    display: flex;
+    flex-wrap: nowrap;
```
Comment (siarheikarabko) [inline style.css:422]: `nowrap` is the default value. Please review places where you explicitly set `wrap`

File: style.css [hunk 2]
```diff
@@ -391,7 +531,9 @@
     overflow-wrap: break-word;
 }
 
-@media (min-width: 655px) {}
+.author-label__link {
```
Comment (siarheikarabko) [inline style.css:534]: Author info looks strange on mobile devices, please review the styles.

File: style.css [hunk 3]
```diff
@@ -237,6 +250,11 @@
...
 }
 
+.gallery-controls__controller:hover,
+.gallery-controls__controller:focus {
+    background-color: var(--color11);
+}
+
 .gallery-controls__controller-right {
     margin-left: 15px;
     background-image: url(static/images/arrow-left.png);
```
Comment (siarheikarabko) [inline style.css:260]: Please, review the arrow positions.

File: style.css [hunk 4]
```diff
@@ -237,6 +250,11 @@
     background-position: center;
 }
 
+.gallery-controls__controller:hover,
```
Comment (siarheikarabko) [inline style.css:253]: Please, use `control` instead of `controller`.

File: style.css [hunk 5]
```diff
@@ -347,38 +374,151 @@
...
+}
+
+.subscribe-block__email-field:focus {
+    background-color: var(--color10);
 }
 
-.background-white {
+.subscribe-button {
+    font-weight: 900;
+    width: 20%;
```
Comment (siarheikarabko) [inline style.css:441]: `.subscribe-block__email-field` and `.subscribe-button` both are children of `.subscribe-block__input` and according to the styles should take 120% of the parent width (100% + 20%). Please, review.

File: style.css [hunk 6]
```diff
@@ -347,38 +374,151 @@
...
-    width: 60%;
-    height: 110px;
-    margin: 10px 0;
-    background-color: lightgreen;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.media-circle {
```
Comment (siarheikarabko) [inline style.css:515]: Consider changing colors if the parent link in a hover/focus state.

Comment (siarheikarabko) [reply style.css:515]: Please, add the same to `Site Name` link as well.

File: index.html [hunk 7]
```diff
@@ -238,29 +230,73 @@ <h4 class="comments__title">
...
+                <p class="section__description section--color-white">
                     Donec odio uisque volutpat mattis eros ullam malesuada erat ut turpis.
                 </p>
             </div>
-            <div class="subscribe-block content-container">
-                <div class="subscribe-block__container"></div>
-                <div class="subscribe-block__container"></div>
-            </div>
+            <ul class="subscribe-block content-container">
+                <li class="subscribe-block__container">
```
[the remark this answers is not in the document]
Comment (kovpaka) [reply index.html:239]: Fixed

File: index.html [hunk 8]
```diff
@@ -238,29 +230,73 @@ <h4 class="comments__title">
...
             <div class="content-container">
-                <div class="media-links">
-                    <div class="media-links__link"></div>
-                    <div class="media-links__link"></div>
-                    <div class="media-links__link"></div>
-                </div>
+                <ul class="media-links">
+                    <li>
+                        <a class="media-links__link" href="###">
+                            <div class="media-circle">
```
[the remark this answers is not in the document]
Comment (kovpaka) [reply index.html:262]: Thanks, fixed

File: style.css (hunk 1 above)
Comment (kovpaka) [reply style.css:422]: Fixed

File: style.css (hunk 2 above)
Comment (kovpaka) [reply style.css:534]: Fixed

File: style.css (hunk 3 above)
Comment (kovpaka) [reply style.css:260]: Fixed

File: style.css (hunk 4 above)
Comment (kovpaka) [reply style.css:253]: Fixed

File: style.css (hunk 6 above)
Comment (kovpaka) [reply style.css:515]: Done.

**Pull request merged**